Survival-DEV/blog-backend
# Blog-10: create comment table

### :dart:  What is been done in this PR:
**endpoint** : `api/comments`
- [x] Create comments entity
- [x] Create CRUD ops, interface & DTO models

**Relates to #10**

----
### Output Response:
 > 
 ```json
{
  {
        "id": "bece8039-422c-466c-ba92-692eddce730d",
        "title": "What the heck is that",
        "is_published": true,
        "created_at": "2021-06-30T14:04:18.847Z",
        "updated_at": "2021-06-30T14:04:18.847Z",
        "content": "This post is wickedly holds unrealistc facts and misleading information stop spreading the stupidity among the inter-net",
        "parent_id": null
    }
}
 ```
 -----

### :compass:  How to test it up
Steps to test my PR:
1. `npm ci`
2. make sure to setup your `DATABASE_URL`& `PORT` in your `.env.development` file
6. `nest start`
4. the active routes are-with their CRUD ops- :
   - `api/comments` , `api/comments/:id` & `api/blogs/:blogId`
